Corona-Warn-App/src/main/res/values-en/strings.xml

```diff
@@ -535,7 +535,7 @@
     <!-- XHED: Page title for terms of use information page, also menu item / button text -->
     <string name="information_terms_title">"Terms of Use"</string>
     <!-- XHED: Page headline for terms of use information page -->
-    <string name="information_terms_headline">"Content"</string>
+    <string name="information_terms_headline">"Inhalt"</string>
```

Comment:
This file is /values-en/strings.xml -> the german "Inhalt" is wrong

Reply:
fixed